src/plugins/unified_doc_viewer/public/components/field_name/field_name.tsx

```diff
@@ -62,7 +62,7 @@ export function FieldName({
             position="top"
             delay="long"
             content={i18n.translate(
-              'discover.fieldChooser.discoverField.multiFieldTooltipContent',
+              'unifiedDocViewer.fieldChooser.discoverField.multiFieldTooltipContent',
```

Comment:
QQ: I see some components need refactoring or are blocked from being moved to a package, so this can be follow up tasks. But fieldname could already be used, right?